### PR TITLE
Updated README and INSTALL md files

### PR DESCRIPTION
**Pull request by StevenMercurio**
Made updates to the README.md install to point to INSTALL.md and edited the INSTALL.md

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -146,44 +118,50 @@ If you follow the below steps on greed field, configured, Red Hat CloudForms app
...
+    *   Name: location
+    *   Description: Location
+    *   Long Description: Location
+    *   Show in Console: Yes
+    *   Single Value: No
+    *   Capture C & U Data by Tag: No
 
 ### Operating System Tag Category
-
+This is a new tag that will need to be created and ipopulated with what operating systems are supported from within CFME
```
Comment (curt-matthews) [inline INSTALL.md]: This is a new tag category created and populated with operating systems values appropriate to your organization

File: INSTALL.md [hunk 2]
```diff
@@ -146,44 +118,50 @@ If you follow the below steps on greed field, configured, Red Hat CloudForms app
...
+This tag will be used to tag provider images with the operating system that the image is configured to deploy
```
Comment (curt-matthews) [inline INSTALL.md]: This tag will be used to tag provider images with the operating system value that the image is configured to deploy

File: README.md [hunk 3]
```diff
@@ -31,10 +31,4 @@ These ManageIQ atuomate domains must also be installed for this datastore to fun
...
 
 # Install
-0. Install dependencies
-1. Automate -> Import/Export
-2. Import Datastore via git
-3. Git URL: `https://github.com/rhtconsulting/miq-RedHat-Satellite6.git`
-4. Submit
-5. Select Branc/Tag to syncronize with
-6. Submit
+Please see the [INSTALL.md](https://github.com/RedHatOfficial/miq-RedHat-Satellite6/blob/master/INSTALL.md) file for install instructions
```
Comment (curt-matthews) [inline README.md]: /install/installation/

**Comment by StevenMercurio** [comment]
Creating new request with addl changes and now a more scripted method of install.

**Pull request closed without merging**